command/exec.go

```diff
@@ -206,6 +207,10 @@ func (c *subprocessExec) Execute(ctx context.Context, comm client.Communicator,
 		return errors.WithStack(err)
 	}
 
+	logger.Execution().WarningWhenf(filepath.IsAbs(c.WorkingDir),
```

Comment:
It feels unnecessary to log this since this will potentially be added to every task in certain projects, and we either handle it silently with this change or the user will notice that their command doesn't work at all

Reply:
I think it would increase user confusion, since presumably they put the workdir as an absolute path which will work as they expect, but they'll get a warning message